### ISSUE TITLE
Error:transformClassesWithRetrolambdaForRelease

### ISSUE THREAD

**Issue by flyingun** [evant/gradle-retrolambda#231]
Hi,
i got this error:
Error:Execution failed for task ':PrjSDK:transformClassesWithRetrolambdaForRelease'.
> Process 'command 'D:\XXXX\XXXXX\XXXX\jre\bin\java.exe'' finished with non-zero exit value -1073740791
**build.gradle**:
dependencies {
        classpath 'com.android.tools.build:gradle:2.2.3'
        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
        classpath 'me.tatarka:gradle-retrolambda:3.5.0'
    }

**Comment by dtukov-qulix** [comment]
Also got this error. Seems it is related to proguard because the error is only thrown for module with 'minifyEnabled true'.

**Comment by Doko-Demo-Doa** [comment]
I also got this error, but in debug ('transformClassesWithRetrolambdaForDevDebug').

No luck still after trying with google.

**Comment by colinrtwhite** [comment]
It's likely related to this issue: https://github.com/orfjackal/retrolambda/issues/121

**Comment by flyingun** [comment]
Never mind, guys.Everything works fine after reinstalling my OS.........

**Comment by Doko-Demo-Doa** [comment]
Found it.

It seems that because of latest nVIDIA driver. Rolling it back resolves the issue.

**Comment by flyingun** [comment]
Oh, yeah? I don't know if that works, cause I had reinstalled my OS. Maybe next time!

**Comment by Doko-Demo-Doa** [comment]
Yeah, to be more precise. It's 378.49 that caused the trouble. I didn't test with newer version. You can find version history here:

http://www.nvidia.com/download/find.aspx

Just rolling it back to 376.33 or 376.19 (or maybe earlier).

**Comment by flyingun** [comment]
Now, I'm using the version 378.66, it works just fine.
